src/renderer/views/Channel/Channel.js

```diff
@@ -29,6 +30,7 @@ export default defineComponent({
     'ft-channel-bubble': FtChannelBubble,
     'ft-loader': FtLoader,
     'ft-element-list': FtElementList,
+    'ft-community-post': FtCommunityPost,
```

Comment:
This doesn't seem to be used anymore and is likely left over from previous commits in this PR.